Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Short.mo

```diff
@@ -1,9 +1,6 @@
 within Modelica.Magnetic.QuasiStatic.FundamentalWave.Components;
 model Short "Short connection"
   extends Interfaces.TwoPortElementary;
-equation
-  connect(port_p, port_n) annotation (Line(points={{-100,0},{-1,0},{-1,0},{
-          100,0}}, color={255,170,85}));
```

Comment:
See previous comment on short.

Reply:
Re-introduced connection in 24a0c38fddd9acb50aaa994f21dc5475546834f3